### ISSUE TITLE
(Bug) Remove custom icon makes callout icons odd

### ISSUE THREAD

**Issue by Ninccino** [Akifyss/obsidian-border#147]
Steps to reproduce,
1. Components -> File explorer -> Icon -> Turn on "Remove custom icon."
2. Create a callout.
3. While in live preview, click anywhere on the doc.

Obsidian 1.5.3 - With and without plugins:
![Screenshot 2024-01-14 220855](https://github.com/Akifyss/obsidian-border/assets/68035002/bdfefb98-bf3a-4650-a9b4-08046c89837c)

It's pretty slight, but this is what it's supposed to look like:
![image](https://github.com/Akifyss/obsidian-border/assets/68035002/6f7c8c43-e388-464b-8fb5-8b126d51cbb3)

**Comment by Akifyss** [comment]
This is very strange because this feature has nothing to do with callout icons, and it also works fine in my environment:
![1](https://github.com/Akifyss/obsidian-border/assets/23567154/c85e2b92-64b6-4f8f-a5b5-ecf75a964edd)

Can you test it again in a sandbox environment and provide some videos to help locate the issues more quickly?

**Comment by Ninccino** [comment]
I got the issue to show up without themes or any plugins.  I kinda forgot I'd had this issue before on another theme anyways.

As requested, I tried to reproduce it in sandbox, and it failed at first.  But then it happened without the option being toggled, then without plugins, then without themes?

The only consistent thing about the issue is it only happens when you click on a heading size of 1-2, but if you click before the # it won't change.  My theory is it only happened every line because of a plugin I had called Show Whitespace.  Except that plugin fixed the issue when I turned it on in the sandbox?
Maybe my Obsidian got corrupted?

I did end up taking a video for you if you're interested.  It's without the theme tho...  And I think Github doesn't like the format despite allowing it, so it might not actually work.

https://github.com/Akifyss/obsidian-border/assets/68035002/c702f3ca-2d2a-4375-b52a-1d584c248c0c
